tests/Feature/GraphQL/Mutations/CreateListingTest.php

```diff
@@ -75,9 +73,11 @@ public function test_it_can_skip_validation(): void
                 'amount' => fake()->numberBetween(1, 1000),
                 'price' => fake()->numberBetween(1, 1000),
                 'salt' => fake()->text(10),
-                'auctionData' => [
-                    'startBlock' => fake()->numberBetween(1011, 5000),
-                    'endBlock' => fake()->numberBetween(5001, 10000),
+                'listingData' => [
+                    'type' => ListingType::AUCTION->name,
+                    'auctionParams' => ['startBlock' => fake()->numberBetween(1011, 5000),
+                        'endBlock' => fake()->numberBetween(5001, 10000),
+                    ],
```

Comment:
```suggestion
                'listingData' => [
                    'type' => ListingType::AUCTION->name,
                    'auctionParams' => [
                        'startBlock' => fake()->numberBetween(1011, 5000),
                        'endBlock' => fake()->numberBetween(5001, 10000),
                    ],
```
Just some formatting.